src/Kiota.Builder/Kiota.Builder.csproj

```diff
@@ -3,7 +3,7 @@
     <LangVersion>Latest</LangVersion>
     <Nullable>enable</Nullable>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <TargetFramework>net8.0</TargetFramework>
+    <TargetFrameworks>net8.0;net9.0;</TargetFrameworks>
```

Comment:
why keep both targets if our releasing pipeline only publishes net9?

Reply:
(I'm unsure about this tbh. I still need to validate this first)
Given that we'll still publish this as a nuget tool, won't having the target as only net 9 break the users running the tool using net8.0? 
In my head, the two targets are not for the purposes for the binaries but really for the nuget tool for users in net8.0 and 9.0.

Reply:
in that case the -f net9.0 in the azure devops pipeline needs to be removed I believe.

Reply:
thanks for clarifying my confusion between the binaries and the nuget package!